content/blog/elm-at-noredink/index.md

```diff
@@ -0,0 +1,445 @@
+---
+title: "Elm at NoRedInk"
+date: "2021-02-15T12:00:00.000Z"
+description: How to organize 500K lines of Elm.
+---
+
+At [NoRedInk](https://www.noredink.com) we have one of the largest Elm apps
+in the world. It serves millions of teachers and students every day and our
+frontend is exclusively written in Elm. In this post, we will explore the
+structure of our application and the patterns that we use to stay sane.
+
+One of the most common questions I get about Elm is "But does it scale? And
+if so, how". Well, I hope that this post can become a comprehensive answer
+to that question.
+
+## Some stats first
+
+We have a glorious monorepo which contains all the services that power our
+infrastructure. Our main application (aka the `monolith`) is written in
```

Comment:
💭  even though we are used to it, I think the name `monolith` is a little misleading in the context of a monorepo of services. also, I would expect it to become less of a monolith as we transition to Haskell. how about using something like "our biggest application"?

Reply:
Yeah, good point! I wanted to mention `monolith` because it appears in the bash commands later.